tests/test_unit/test_cdf_tk/test_loaders/test_fieldops_loaders.py

```diff
@@ -0,0 +1,61 @@
+from cognite_toolkit._cdf_tk.client.data_classes.apm_config_v1 import (
+    APMConfigWrite,
+    FeatureConfiguration,
+    ResourceFilters,
+    RootLocationConfiguration,
+    RootLocationDataFilters,
+)
+from cognite_toolkit._cdf_tk.loaders import (
+    AssetLoader,
+    DataSetsLoader,
+    GroupResourceScopedLoader,
+    InfieldV1Loader,
+    SpaceLoader,
+)
+
+
+class TestInfieldV1Loader:
+    def test_dependent_items(self) -> None:
```

Comment:
Note that the other methods are tested through other tests when adding the test data.